app/views/admin/users/index.html.erb

```diff
@@ -1,5 +1,32 @@
 <p id="notice"><%= notice %></p>
 
 <h1>Users</h1>
-<h3>Users are listed under their facilities</h3>
-<%= link_to 'List facilites', admin_facilities_path %>
\ No newline at end of file
+
+<table class="table">
+  <thead class="thead-light">
+  <tr>
+    <th>Name</th>
+    <th>No. of Facilities</th>
+    <th colspan="2"></th>
+  </tr>
+  </thead>
+
+  <tbody>
+  <% @users.each do |user| %>
+    <tr>
+      <td><%= link_to user.full_name, [:admin, user] %></td>
+      <td><%= user.facilities.count %></td>
+      <% if policy(user).edit? %>
+        <td><%= link_to 'Edit', edit_admin_user_path(user) %></td>
+      <% end %>
+      <% if policy(user).destroy? %>
+        <td><%= link_to 'Delete', [:admin, user], method: :delete, data: { confirm: 'Are you sure?' } %></td>
+      <% end %>
+    </tr>
+  <% end %>
+  </tbody>
+</table>
+
+<% if policy(User).new? %>
+  <%= link_to 'New User', new_admin_user_path, class: "btn btn-sm btn-primary" %>
```

Comment:
I haven't removed this feature yet, because I'm not sure that we don't need it anymore. The user can be approved at the the time of creation, and then they can log in to use the app.
I was thinking we would remove this feature in a future PR.